subprojects/packagefiles/google-highway/meson.build

```diff
@@ -0,0 +1,155 @@
+project(
+  'highway',
+  'cpp',
+  version: '1.2.0',
+  license: 'Apache-2.0',
+)
+
+highway_sources = files(
+    'hwy/abort.cc',
+    'hwy/aligned_allocator.cc',
+    'hwy/nanobenchmark.cc',
+    'hwy/per_target.cc',
+    'hwy/print.cc',
+    'hwy/targets.cc',
+    'hwy/timer.cc',
+)
+
+if host_machine.system() == 'windows'
+  add_project_arguments('-DNOMINMAX', language: 'cpp')
+endif
+is_shared = (get_option('default_library') != 'static')
+args = is_shared ? ['-DHWY_SHARED_DEFINE', '-Dhwy_EXPORTS'] : ['-DHWY_STATIC_DEFINE']
```

Comment:
Don't do this, it breaks `default_library == 'both'`, instead add:
```
    cpp_static_args : ['-DHWY_STATIC_DEFINE'],
    cpp_shared_args : ['-DHWY_SHARED_DEFINE', '-Dhwy_EXPORTS'],
```

to the `lib_highway` below. This will do the right thing regardless of what `get_option('default_library')`